### DISCUSSION TITLE
Auto "wake-on-lan" or auto-connect to Mount

### DISCUSSION THREAD

**Discussion by rkas12** [mworion/MountWizzard4#242] [Feature Request] [answered]
Good evening,

I hope you are doing well.

In fact, this is not an issue but more a request. Is there a simple way to automatically connect the mount when MW4 starts ? 

Thank you for your kind followup.

Best regards,
A.

**Comment by mworion** [comment]
Hi,
I moved to to discussion as it is a feature request. For understanding, you would like to have a WOL call to the mount once you start MW4?
Michel

**Comment by rkas12** [comment]
Hi Michel,

Thank you for your follow-up. 

Exactly, I don't know how much effort it will require to have this up and running, but it would be nice. 

Best regards,
Aygen

**Comment by Ruediger3** [comment]
Hello both,
please only as an option, which you can deactivate. 

Thanks!
Rüdiger

**Comment by mworion** [comment, marked as answer]
OK,understood
I put it on the v3.0 wishlist
Michel

**Comment by rkas12** [comment]
Thank you, Michel ! 

Meanwhile, I was wondering whether I could use, for instance, windows Task Scheduler with an argument to a) launch MW4 (this, I am sure I can) and then b)execute a wolf ?

**Comment by mworion** [comment]
Hi Aygen,
sure, this is another possibility. But implementing took only a glimpse, will be available from v3.0 on:
![2022-05-29-18-13-46-MountWizzard4_-_v3 0 0b0](https://user-images.githubusercontent.com/6632980/170885430-3b6fc994-671d-4661-b2cf-cc7a91ef52b0.png)

![2022-05-29-18-14-38-Messages](https://user-images.githubusercontent.com/6632980/170885411-039bf66e-af33-4090-a173-cb9bdb90bf39.png)
Michel

**Comment by rkas12** [comment]
Incredible, I am really looking forward to it since it will make easier my
perpetual script in Voyager :)
You rock !
Thank you Michel and all the best,
Aygen

**Comment by rkas12** [comment]
Just one question: is that okay if let some Voyager / 10 Micron user know
about this future « wol option » ?

Le dim. 29 mai 2022 à 20:35, Erkaslan Aygen ***@***.***> a
écrit :

> Incredible, I am really looking forward to it since it will make easier my
> perpetual script in Voyager :)
> You rock !
> Thank you Michel and all the best,
> Aygen
>
>

**Reply by mworion** [reply]
No problem.
